drivers/include/at86rf2xx.h

```diff
@@ -63,12 +63,18 @@ extern "C" {
 /* the AT86RF212B has a sub-1GHz radio */
 #define AT86RF2XX_MIN_CHANNEL           (0)
 #define AT86RF2XX_MAX_CHANNEL           (10)
+#ifdef DEFAULT_CHANNEL
+#define AT86RF2XX_DEFAULT_CHANNEL (DEFAULT_CHANNELU)
```

Comment:
Well, no, just a misunderstanding of macro on my side... I removed the added _U_'s. I wanted to keep the initial `(26U)` notation using the defined variable. Maybe using `##` could make it but seems a bit too complex.